v3/src/components/graph/adornments/adornment-models.ts

```diff
@@ -63,45 +63,53 @@ export const AdornmentModel = types.model("AdornmentModel", {
     },
     get isUnivariateMeasure() {
       return false
-    }
-  }))
-  .actions(self => ({
-    setVisibility(isVisible: boolean) {
-      self.isVisible = isVisible
     },
-    updateCategories(options: IUpdateCategoriesOptions) {
-      // derived models should override to update their models when categories change
-    },
-    setCellKey(options: IUpdateCategoriesOptions, index: number) {
+    generateCellKey(options: IUpdateCategoriesOptions, index: number) {
```

Comment:
What about `getCellKey` or even just `cellKey`? 🤷

Reply:
I'm going to go with `cellKey`. I think that makes sense. Thanks.